Plugwise_Smile/Smile.py

```diff
@@ -136,10 +136,11 @@ async def _create_session() -> aiohttp.ClientSession:
         self.dsmrmain_id = None
         self.gateway_id = None
         self.heater_id = None
-        self.hostname =  None
+        self.hostname = None
         self.smile_name = None
         self.smile_type = None
         self.smile_version = ()
+        self.notifications = {}
```

Comment:
No need to have this line because this is repeated in 324.
I moved this line on purpose to 324 to clear `self.notification` when the Plugwise Notification disappears.

Reply:
pylint complained about not initialized in init - as in - all self attributes should be in init.

Reply:
Ok, understood. Having this line there should not cause any issue.